spec/unit/analyzer/resources_analyzer/font_resources_analyzer_spec.rb

```diff
@@ -0,0 +1,22 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe 'XCRes::ResourcesAnalyzer::FontResourcesAnalyzer' do
+
+  def subject
+    XCRes::ResourcesAnalyzer::FontResourcesAnalyzer
+  end
+
+  before do
+    @analyzer = subject.new
+    @analyzer.logger = stub('Logger', :log)
+  end
+
+  describe '#analyze' do
+    # TODO
```

Comment:
I know that the other resource analyzers are not all fully covered and there are still open TODOs, but I'd prefer to not add additional ones. I think adding a positive test for each of both methods on base of the fixture project would be fine here.